### ISSUE TITLE
Organising and documenting stuff

### ISSUE THREAD

**Issue by melisandeteng** [cc-ai/kdb#134]
Where would be a good place to keep "general" information on github? 
For example, I'm thinking about : 
1) where the different datasets are on the cluster, what they correspond to. (having files named trainA.txt and trainAB.txt and trainAC.txt is not very helpful... ) 
2) which .txt files to use in the MaskGenerator
3) how to make cityscapes labels match with our simulated world labels. 

I guess 2) could be in the MaskGenerator README, and 1) somewhere in the kdb (where ? )
But what about 3) ? We have a lot of such information scattered in README's of different repos and it's a nightmare to fetch information (plus it relies on human memory of the people who wrote the readme's) but it might be somewhat too specific to be in the kdb ? Should I create a new repo for datasets utils ? 

Let me know what you think !

**Comment by vict0rsch** [comment]
I agree with you. That's what the kdb is intended for, even though we don't leverage it to its full potential :p  So either these become issues which we keep open or you add `md` files to https://github.com/cc-ai/kdb/tree/master/resources

<blockquote><img src="https://avatars3.githubusercontent.com/u/48456149?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/cc-ai/kdb">cc-ai/kdb</a></strong></div><div>AI can help visualizing the impact of climate change. This is an open forum to share our work - cc-ai/kdb</div></blockquote>

**Comment by melisandeteng** [comment]
Ok @vict0rsch ! I'll update the `resources` page for the datasets, and I created a new repo for all Simulated dataset related utils : https://github.com/cc-ai/SimData

<blockquote><img src="https://avatars3.githubusercontent.com/u/48456149?s=400&v=4" width="48" align="right"><div><img src="https://github.githubassets.com/favicons/favicon.svg" height="14"> GitHub</div><div><strong><a href="https://github.com/cc-ai/SimData">cc-ai/SimData</a></strong></div><div>About our simulated world, how read depth maps,  make segmentation labels match with other datasets, and more  - cc-ai/SimData</div></blockquote>
